tests/bats/run_container_with_rafs_and_compile_linux.bats

```diff
@@ -1,7 +1,7 @@
 load "${BATS_TEST_DIRNAME}/common_tests.sh"
 
 setup() {
-	nydus_rafs_image="nydus-anolis-registry.cn-hangzhou.cr.aliyuncs.com/nydus_test/bldlinux:v0.1-rafs-v6-lz4"
+	nydus_rafs_image="docker.io/hsiangkao/bldlinux:v0.1-rafs-v6-lz4"
```

Comment:
I am worried about the QPS limitation from dockerhub which can break the CI test

Reply:
Firstly, the test machine is located in Europe.
Secondly, the other 2 test cases below use the image in dockerhub, which is more stable than this test case which uses the image in hangzhou, based on the previous testing history.
https://github.com/dragonflyoss/image-service/blob/master/tests/bats/run_container_with_rafs.bats
https://github.com/dragonflyoss/image-service/blob/master/tests/bats/run_container_with_zran.bats

Reply:
@yqleng1987 I just uploaded a image to the github repository: `ghcr.io/dragonflyoss/image-service/bldlinux:v0.1-rafs-v6-lz4`, could you try it to be able to use in CI?